cmd/geteduroam/main.go

```diff
@@ -362,6 +379,18 @@ func main() {
 		flag.Usage()
 		return
 	}
+	logLevel := &slog.LevelVar{}
+	opts := &slog.HandlerOptions{
+		Level: logLevel,
+	}
+	slog.SetDefault(slog.New(slog.NewTextHandler(os.Stderr, opts)))
+	if verbose {
+		utils.IsVerbose = true
+	}
+	if debug {
+		logLevel.Set(slog.LevelDebug)
+		utils.PrintLevels()
+	}
```

Comment:
Could we only log to a file and not stdout?

Reply:
verbose to stdout is fine of course, but slog only to a file I was picturing

Reply:
And where should this file be written? CWD?